reactivesocket-cpp/src/ConnectionAutomaton.cpp

```diff
@@ -103,6 +113,7 @@ void ConnectionAutomaton::onSubscribe(Subscription& subscription) {
   connectionInputSub_.reset(&subscription);
   // This may result in signals being issued by the connection in-line, see
   // ::connect.
+  // TODO: implement lease here. For now we will disable flow control
```

Comment:
Lease cannot be implemented here, we must always accept all frames as soon as we can, since some of them do not translate into lease-controlled events (e.g. closing a stream). Please remove.